set.mm

```diff
@@ -319161,6 +319161,52 @@ can be the countable union of rare closed subsets (where rare means
       HZJSDGHZDKHQRSLQRTSABCDEFMNDOP $.
   $}
 
+  ${
+    cssbn.x $e |- X = ( W |`s U ) $.
+    cssbn.s $e |- S = ( LSubSp ` W ) $.
+    cssbn.d $e |- D = ( ( dist ` W ) |` ( U X. U ) ) $.
+    $( Banach space property of a closed subspace: a closed subspace of a
+       normed vector space with a closed scalar field is a Banach space.
+       Remark:  In contrast to ` ClSubSp ` , a closed (compact?) subspace is
+       defined by "a linear subspace in which all Cauchy sequences converge to
+       a point in the subspace".  (Contributed by NM, 10-Apr-2008.)  (Revised
+       by AV, 6-Oct-2022.) $)
+    cssbn $p |- ( ( ( W e. NrmVec /\ ( Scalar ` W ) e. CMetSp /\ U e. S )
+                    /\ ( Cau ` D ) C_ dom ( ~~>t ` ( MetOpen ` D ) ) )
+                  -> X e. Ban ) $=
+      ( cnvc wcel csca cfv ccms 3adant2 adantr cds cxp syl wceq eqid w3a cca wa
+      cmopn clm cdm wss cbn lssnvc cmt cres cngp csubg cnlm nvcnlm nlmngp clmod
+      cbs nvclmod lsssubg sylan subgngp syl2an2r ngpms 3ad2ant3 subgbas sqxpeqd
+      cms ressds reseq12d syl5eq eqcomd cme ngpmet eqeltrd simpr sylanbrc iscms
+      iscmet2 simpl3 resssca simpl2 eqeltrrd isbn syl3anbrc ) DIJZDKLZMJZCBJZUA
+      ZAUBLAUDLZUELUFUGZUCZEIJZEMJZEKLZMJEUHJWJWNWLWFWIWNWHBCDEFGUINOWMEUJJZEPL
+      ZEURLZWSQZUKZWSVHLZJWOWMEULJZWQWJXCWLWFWIXCWHWFDULJZWICDUMLJZXCWFDUNJXDDU
+      ODUPRWFDUQJWIXEDUSBCDGUTVAZCDEFVBVCNZOEVDRWMXAAXBWJXAASWLWJAXAWJADPLZCCQZ
+      UKXAHWJXHWRXIWTWIWFXHWRSWHCXHDEBFXHTVIVEWJCWSWJXECWSSWFWIXEWHXFNCDEFVFRVG
+      VJVKZVLOWMAWSVMLZJZWLAXBJWJXLWLWJAXAXKXJWJXCXAXKJXGXAEWSWSTZXATZVNRVOOWJW
+      LVPAWKWSWKTVSVQVOXAEWSXMXNVRVQWMWGWPMWMWIWGWPSWFWHWIWLVTCWGDEBFWGTWARWFWH
+      WIWLWBWCWPEWPTWDWE $.
+
+    $( Hilbert space property of a closed subspace: a closed subspace of a
+       complex pre-Hilbert space is a complex Hilbert space.  Remarks:  (a) In
+       contrast to ` ClSubSp ` , a closed (compact?) subspace is defined by "a
+       linear subspace in which all Cauchy sequences converge to a point in the
+       subspace".  (b) This theorem does not hold for arbitrary subcomplex
+       (pre-)Hilbert spaces, because the scalar field as restriction of the
+       field of the complex numbers need not be closed.  (Contributed by NM,
```

Comment:
Remove the part before the colon and Remark (a).
... because the scalar field need not be a closed subfield of ` CCfld ` .